packages/mds-logger/index.ts

```diff
@@ -19,9 +19,7 @@ import httpContext from 'express-http-context'
 const logger: Pick<Console, 'info' | 'warn' | 'error'> = console
 type LogLevel = keyof typeof logger
 
-type LogArgs = [message: string, data?: Record<string, unknown> | Error]
-
-const redact = (arg: unknown): any => {
+const redact = (arg: string | Record<string, unknown> | Error | undefined): any => {
```

Comment:
Just curious, what's the inferred return type if you remove the `: any`?

Reply:
It's just an `any`, so doesn't need to be explicit I suppose